gateway/1.5/modules/ROOT/pages/policies-included-traffic-management.adoc

```diff
@@ -0,0 +1,154 @@
+
+== Configuring Policy Parameters
+
+Refer to the following policy definition and table of parameters:
+
+----
+policyRef:
+    name: route-weighted
+  config:
+    routes:
+    - weight: <int> // REQUIRED, max_value=100, min_value=1
+      destinationPath: <string> // OPTIONAL, default: "/"
+      destinationRef: 
+        name: <string> // REQUIRED
+  rules:
+   - path: <URI-regex> // OPTIONAL, Example: (.*)
+     methods: <string> // OPTIONAL, Example: GET|POST
+     headers:
+       <header-name>: <regular-expresion> // OPTIONAL
+     host: <string> // OPTIONAL
+----
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+|===
+| Parameter | Required or Optional | Default Value | Description
+
+| `routes`
+| Required
+| Empty
+| Array of API upstream services. +
+At least one upstream is required.
+
+| `routes.weight`
+| Required
+| N/A
+| The weight of total requests sent to this upstream service. +
+ The percentage of traffic sent to that upstream service is the upstream service's weight divided by total weight of all upstream services. For example, if there are two upstream services with weight `1`, each upstream service receives 50% of the traffic to that route.
+
+| `routes.destinationPath`
+| Required
+| N/A
+| Path of the upstream service
+
+| `routes.destinationRef.name`
+| Required
+| N/A
+| Name of the upstream service as defined in the `Service` configuration resource
+
+| `rules`
+| Optional
+| Empty
+| Array of rulesets for this route. +
+Each route can support multiple rulesets.
+
+| `rules.path`
+| Optional
+| Empty
+| The request path that the route can service. +
+You can define only one “URI Template Regex" path for each ruleset. Only requests with the defined path are sent to this route.
+
+| `rules.methods`
+| Optional
+| Empty
+| Array that defines the types of request methods that the route can service. +
+You can select multiple methods for each ruleset. Only requests that are one of the defined methods are sent to this route. Supported values are `GET`, `POST`, `PUT`, `PATCH`, `DELETE`, `OPTIONS`, `HEAD`, `TRACE`, and `CONNECT`.
+
+| `rules.headers`
+| Optional
+| Empty
+| Array that defines what headers and regular expression value must be present for this route to service the request. +
+Only requests that meet all of the specified header requirements are sent to this route. Additional headers present in the request that are not specifically defined in the rules are ignored.
+
+| `rules.host`
+| Optional
+| Empty
+| The request host URL that the route can service. +
+You can define only one host URL for each ruleset. Only requests made from the defined host are sent to this route.
+
+|===
+
+== How This Policy Works
+
+The Traffic Management policy directs traffic to different sets of upstream services by using different routes with defined rulesets and route order. 
```

Comment:
I think I had a comment about this in the last PR I reviewed for you. It is better to have "route order" first here because otherwise it implies that each route has a route order defined.